### PR TITLE
update cors domain page for wildcard restriction

### PR DESCRIPTION
**Pull request by adamsachs**
Closes [PROD-1203](https://ethyca.atlassian.net/browse/PROD-1203)

### Description of Changes

* [x] update wording to make clear that wildcard syntax is not accepted in cors domains UI
* [x] update screenshot to actually show valid values!

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* [x] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created

[PROD-1203]: https://ethyca.atlassian.net/browse/PROD-1203?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: public/assets/img/configuration/cors_configuration.png
Comment (adamsachs) [inline public/assets/img/configuration/cors_configuration.png:1]: whoops, dimensions are a bit different - is that OK? i just took the screenshot on my local, not sure if we do something more standardized generally...

**Comment by adamsachs** [comment]
wasn't sure who was best to review this - tagged you both @NevilleS and @rsilvery for now, but let me know if someone else should look it over!

just a quick copy update and a screenshot update. happy to adjust as needed, or feel free to adjust to your desire 👍

File: pages/dev-docs/get-started/domains/allowed-domains.mdx [hunk 1]
```diff
@@ -19,7 +19,7 @@ To add a domain to the Fides Cloud allowed list:
 2. From the navigation menu on the left, select **CORS configuration**
 3. Add your CORS domain
 
-<Callout emoji="" type="info"> Note, when adding a domain, you may use a wildcard, such as [*.yourdomain.com](http://*.yourdomain.com), or a fully qualified subdomain.</Callout>
+<Callout emoji="" type="info"> Note, when adding a domain, you may only use valid URLs - wildcards are not supported for security reasons. For advanced usage, wildcard domains can be set up outside of the UI using a separate application configuration property.</Callout>
```
Comment (NevilleS) [inline pages/dev-docs/get-started/domains/allowed-domains.mdx:22]:
```suggestion
<Callout emoji="" type="info"> Note, when adding a domain, you must use a valid URL. Wildcards, such as *.yourdomain.com, are not supported for security reasons. To request a wildcard domain, please contact support for assistance.</Callout>
```

File: public/assets/img/configuration/cors_configuration.png
Comment (NevilleS) [reply public/assets/img/configuration/cors_configuration.png:1]: Yup, that looks good! No worries.

**Review by NevilleS — APPROVED** [review]
Quick nitpick on the language, and then good to go 👍

**Pull request merged**